homeassistant/components/media_player/cast.py

```diff
@@ -216,13 +213,15 @@ def _async_create_cast_device(hass: HomeAssistantType,
     if not isinstance(config, list):
         config = [config]
 
-    await asyncio.wait([
+    done, pending = await asyncio.wait([
         _async_setup_platform(hass, cfg, async_add_entities, None)
         for cfg in config])
```

Comment:
Isn't `pending` always empty here? From what I understand from the docs, `pending` contains a set of tasks that are not done when the *timeout* occurs. But there is no timeout configured here...

Also, it would be good to cancel all tasks in `pending`.

Reply:
Removed pending